drop-storage/Cargo.toml

```
@@ -14,12 +14,10 @@ serde = { workspace = true }
 uuid = { workspace = true }
 chrono = { version = "0.4.26", default-features = false, features = ["std"] }
 url = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
```

```suggestion
tokio = { workspace = true }
```
The `rt-multi-thread` features is already enable in workspace